### ISSUE TITLE
Implement privacy (exclude by domain and regex)

### ISSUE THREAD

**Issue by Kerrick** [Kerrick/Mostly-Harmless#2]
I really need to implement privacy controls. That way, users can exclude certain pages from being looked up by Mostly Harmless. The user will get two textareas:  one for domains, and one for regular expressions. If a page is on a listed domain, or the URL matches a listed regex pattern, Mostly Harmless will be deactivated for that page.
### Default Domains

```
secure.ingdirect.com
chaseonline.chase.com
online.wellsfargo.com
```
### Default Regular Expressions

```
chrome://.*
chrome-extension://.*
view-source://.*
ftp://.*
https?://www.reddit.com/r/(\w|\+)+/?$
https?://www.reddit.com/?$
https?://www\.google\.com/search.*
https?://search\.yahoo\.com/search.*
https?://www\.bing\.com/search.*
```

**Comment by Kerrick** [comment]
I made some changes to [fancy-settings](https://github.com/frankkohlhepp/fancy-settings) to add support for `textarea`s, and am currently waiting on [pull request 7](https://github.com/frankkohlhepp/fancy-settings/pull/7).

**Comment by Kerrick** [comment]
I've fixed this in commit [1aeac5f](https://github.com/Kerrick/Mostly-Harmless/commit/1aeac5fee00cbcb4d875c95beba13f60434b9bbd).
